src/Emogrifier/HtmlProcessor/HtmlPruner.php

```diff
@@ -36,7 +36,7 @@ public function removeInvisibleNodes()
         /** @var \DOMNode $node */
         foreach ($nodesWithStyleDisplayNone as $node) {
             $parentNode = $node->parentNode;
-            if ($parentNode !== null && \is_callable([$parentNode, 'removeChild'])) {
+            if ($parentNode !== null) {
```

Comment:
I think the comment is incorrect. Either `$node` is `null` and hence does not have a `removeChild` method, or it is non-`null` (i.e., a `DOMNode` instance), and and then it will have a `removeChild` method, and that method on that object will be callable. Or am I missing something?

Reply:
I've removed the comment now.

Reply:
> I think the comment is incorrect.

I think you are right.  It has been like that ever since Emogrifier was put in GitHub, so we don't have an issue to reference for why the `is_callable` may be needed.

I added locally a couple of tests and couldn't cause a failure without `is_callable`:
```php
    /**
     * @test
     *
     * @param string $displayNone
     *
     * @dataProvider displayNoneDataProvider
     */
    public function removeInvisibleNodesRemovesNodesAndChildNodesWithDisplayNone($displayNone)
    {
        $subject = HtmlPruner::fromHtml(
            '<html><body><div style="' . $displayNone . '"><div style="' . $displayNone . '"></div></div></body></html>'
        );

        $subject->removeInvisibleNodes();

        self::assertNotContains('<div', $subject->render());
    }

    /**
     * @test
     *
     * @param string $displayNone
     *
     * @dataProvider displayNoneDataProvider
     */
    public function removeInvisibleNodesRemovesNodesAndGrandchildNodesWithDisplayNone($displayNone)
    {
        $subject = HtmlPruner::fromHtml(
            '<html><body><div style="' . $displayNone . '"><div><div style="' . $displayNone
            . '"></div></div></div></body></html>'
        );

        $subject->removeInvisibleNodes();

        self::assertNotContains('<div', $subject->render());
    }
```

And a search for `php DOMNode "removeChild" "is_callable"` has Emogrifier in most of the top results, so it doesn't look like there's an issue anyone else has had to deal with similarly.